src/server/errors.rs

```diff
@@ -96,9 +95,8 @@ quick_error!{
             from()
             display("{:?}", err)
         }
-        EndPointStopped(err: ScheduleError<EndPointTask>) {
-            from()
-            display("{:?}", err)
+        EndPointStopped { // TODO
```

Comment:
@BusyJay Do you have some ideas to work with this? For now it is `ScheduleError<EndPointTask<T>>`.

Reply:
Is the value used? Can a generic type be added to `Error`? Or can you add a type for each T explicitly. Or move `EndPointStopped` out of enum error.

Reply:
Thanks! I use a solution to manually implement `From` for `E: Engine`.